examples/CMakeLists.txt

```
@@ -891,7 +891,22 @@ add_impactx_test(spectrometer.py
     OFF  # no plot script yet
 )
 
-# Dogleg ########################################################
+# Chicane with CSR ###########################################################
+#
+add_impactx_test(chicane_csr
+    examples/chicane/input_chicane_csr.in
+      OFF  # ImpactX MPI-parallel
+    examples/chicane/analysis_chicane_csr.py
+    OFF  # no plot script yet
+)
+add_impactx_test(chicane_CSR.py
```

Just for consistency in naming:
```suggestion
add_impactx_test(chicane_csr.py
```

---

It looks like something broke due to the recent GPU+MPI support changes.  8/16 of the tests now fail, and the output diagnostics grow out of control.  The last verified working version was at the time of **Aug. 14 commit**.  The problem does not appear to be due to recent changes to WakePush.cpp.

---

On CI, only
```
The following tests FAILED:
	190 - chicane_csr.analysis (Failed)
	192 - chicane_CSR.py.analysis (Failed)
```
seem to fail. But rolling back to August 14th had these tests commented out, I'll check if they passed back then.

---

I rolled back to August 14 locally and `chicane_csr.analysis` passes but `chicane_csr.py.analysis` does not.

---

Ok found it, I broke it in the `GPU support` commit
https://github.com/ECP-WarpX/impactx/pull/627/commits/610689ae206173bc9bd0a72711cae1f475497998

---

I found a few smaller issues and reverted a few key parts in that commit but could not yet nail down the root cause of the issue. we could printf tomorrow the array values at each step of the wakefield calculation to see which one goes off.